src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40 * days;
+  const shortPeriod = 3;
+  const longPeriod = 7;
+  const discountSmall = 20;
+  const discountBig = 50;
+
+  if (days < shortPeriod) {
+    return rentCost;
+  }
+
+  if (days >= shortPeriod && days < longPeriod) {
+    return rentCost - discountSmall;
+  }
+
+  if (days >= longPeriod) {
+    return rentCost - discountBig;
+  }
```

Comment:
Unnecessary `if` statement
```suggestion

  return rentCost - discountBig;
```